### PR TITLE
columnSelect: select all

### LINKED ISSUE #327: Data export: add select all to Column

### PR DESCRIPTION
**Pull request by sorja**
Resolves #327

### CODE REVIEW

File: webapp/app/dataExport/components/columnSelect/columnSelect.less [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.export-column-select {
```
Comment (minotogna) [inline webapp/app/dataExport/components/columnSelect/columnSelect.less:1]: don't we have a similar case in country selection?

Comment (sorja) [reply webapp/app/dataExport/components/columnSelect/columnSelect.less:1]: yes

Comment (minotogna) [reply webapp/app/dataExport/components/columnSelect/columnSelect.less:1]: can we generalize the style then?

Comment (sorja) [reply webapp/app/dataExport/components/columnSelect/columnSelect.less]: yes

**Pull request merged**